CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ this project uses date-based 'snapshot' version identifiers.
 
 ## [Unreleased]
 
+### Added
+- Force non-Windows diff program to consider all files to be text files
+- Extend PDF-based tests to `dvips`
```

Comment:
Does `l3build` test changes need to be listed in CHANGELOG?

Reply:
I'm not sure what you mean - that the change may require `.tlg` updates?

Reply:
the changes only involving "using l3build to test l3build itself"